pandas/tests/io/excel/test_readers.py

```diff
@@ -571,11 +577,17 @@ def test_reader_dtype_str(self, read_ext, dtype, expected):
         actual = pd.read_excel(basename + read_ext, dtype=dtype)
         tm.assert_frame_equal(actual, expected)
 
-    def test_dtype_backend(self, read_ext, dtype_backend):
+    def test_dtype_backend(self, request, engine, read_ext, dtype_backend):
         # GH#36712
         if read_ext in (".xlsb", ".xls"):
             pytest.skip(f"No engine for filetype: '{read_ext}'")
 
+        # GH 54994
+        if engine == "calamine" and read_ext == ".ods":
+            request.node.add_marker(
+                pytest.mark.xfail(reason="OdsWriter produces broken file")
+            )
```

Comment:
Sounds like we should raise an issue?

Reply:
* https://github.com/pandas-dev/pandas/issues/54994
* https://github.com/pandas-dev/pandas/pull/54996